models/productDao.js

```diff
@@ -15,15 +13,55 @@ const getProducts = async (offset, limit, gender) => {
        FROM products AS p
        INNER JOIN sub_categories AS sc ON p.sub_category_id = sc.id
        INNER JOIN main_categories AS mc ON sc.main_category_id = mc.id
-       WHERE mc.name = ? AND p.id LIMIT ?,?`, [gender, numberOffset, numberLimit]
+       WHERE mc.name = ? AND p.id LIMIT ?,?`, [gender, offset, limit]
     );
   } catch (err) {
-    const error = new Error("INVALID_DATA_INPUT");
+    const error = new Error(err.message);
     error.statusCode = 500;
     throw error;
   }
 };
 
+const productDetails = async (productId) => {
```

Comment:
함수명을 좀 더 명확하게 지어주면 더 좋을 것 같아요!

```suggestion
const getProductDetails = async (productId) => {
```